tests/common/snappi/common_helpers.py

```diff
@@ -656,6 +764,11 @@ def enable_packet_aging(duthost):
         duthost.command("docker cp /tmp/packets_aging.py syncd:/")
         duthost.command("docker exec syncd python /packets_aging.py enable")
         duthost.command("docker exec syncd rm -rf /packets_aging.py")
+    elif isBroadcomDevice(duthost):
```

Comment:
You should modify the comment "Enable packet aging feature (only on MLNX switches)"

Reply:
this has been resolved